src/main/resources/plugin.yml

```
@@ -125,3 +125,41 @@ commands:
     description: アイテム受取ボックスを表示します
   addreceive:
     description: アイテム受取ボックスに追加します
+
+  #BirthdayCard
+  birthday:
+    description: お誕生日カードメニューを開きます
+
+permissions:
+  # UniverseCoreV2
+  space.yurisi.universecorev2.interactevent.enchanttable:
+    default: op
+  space.yurisi.universecorev2.giveu:
+    description: op command
+    default: op
+
+  # SubPlugins
+  ## LevelSystem
+  space.yurisi.levelsystem.addexp:
+    description: op command
+    default: op
+
+  ## UniverseEconomy
+  space.yurisi.universeeconomy.*:
+    description: default
+    default: true
+    children:
+      space.yurisi.universeeconomy.money: true
+      space.yurisi.universeeconomy.mymoney: true
+      space.yurisi.universeeconomy.pay: true
+      space.yurisi.universeeconomy.seemoney: true
+  space.yurisi.universeeconomy.setmoney:
+    description: op command
+    default: op
+  space.yurisi.universeeconomy.addmoney:
+    description: op command
+    default: op
+  space.yurisi.universeeconomy.reducemoney:
+    description: op command
+    default: op
```

```suggestion

  # BirthdayCard
  birthday:
    description: お誕生日カードメニューを開きます
```

たつお, コンフリクト解消するときはちゃんと main の変更確認してね ( #156 で `permissions` の部分は全削除してる)